src/vector/geometry.rs

```diff
@@ -213,27 +213,27 @@ impl Geometry {
         let mut y: c_double = 0.;
         let mut z: c_double = 0.;
         let mut m: c_double = 0.;
-        unsafe { gdal_sys::OGR_G_GetPointZM(self.c_geometry(), index, &mut x, &mut y, &mut z, &mut m) };
+        unsafe {
+            gdal_sys::OGR_G_GetPointZM(self.c_geometry(), index, &mut x, &mut y, &mut z, &mut m)
+        };
         (x, y, z, m)
     }
 
-    /// Appends all points of a line string to `out_points`. 
+    /// Appends all points in the geometry to `out_points`, as XYZ.
     ///
-    /// Only wkbPoint[X], wkbLineString[X] or wkbCircularString[X] may alter `out_points`. Other geometry types will silently do nothing, see
-    /// [`OGR_G_GetPointCount`](https://gdal.org/en/stable/api/vector_c_api.html#_CPPv419OGR_G_GetPointCount12OGRGeometryH)
-    pub fn get_point_vec(&self, out_points: &mut Vec<(f64, f64, f64)>) -> usize {
+    /// For some geometry types, like polygons, that don't consist of points, `out_points` will not be modified.
+    pub fn get_points(&self, out_points: &mut Vec<(f64, f64, f64)>) -> usize {
         // Consider replacing logic with
         // [OGR_G_GetPoints](https://gdal.org/en/stable/api/vector_c_api.html#_CPPv415OGR_G_GetPoints12OGRGeometryHPviPviPvi)
         let length = unsafe { gdal_sys::OGR_G_GetPointCount(self.c_geometry()) };
         out_points.extend((0..length).map(|i| self.get_point(i)));
         length as usize
     }
 
-    /// Appends all points of a line string to `out_points`. 
+    /// Appends all points in the geometry to `out_points`, as XYZM.
     ///
-    /// Only wkbPoint[X], wkbLineString[X] or wkbCircularString[X] may alter `out_points`. Other geometry types will silently do nothing, see
```

Comment:
I rephrased this, since it wasn't exactly correct (MultiPoint has points, for example).